src/store/wizardSlice.ts

```diff
@@ -190,6 +224,24 @@ export const wizardSlice = createSlice({
     ) => {
       state.registration.activationKey = action.payload;
     },
+    changeOscapProfile: (
+      state,
+      action: PayloadAction<DistributionProfileItem | undefined>
+    ) => {
+      state.profile = action.payload;
+    },
+    changeOscapProfileName: (
+      state,
+      action: PayloadAction<string | undefined>
+    ) => {
+      state.openScap.profileName = action.payload;
+    },
+    changeOscapProfileDescription: (
+      state,
+      action: PayloadAction<string | undefined>
+    ) => {
+      state.openScap.profileDescription = action.payload;
+    },
```

Comment:
Are you using the functions to change the values somewhere? The structure seems to stay empty even when a profile is selected

![image](https://github.com/RedHatInsights/image-builder-frontend/assets/49452678/cb1c6fd7-9949-4824-bbd6-82442664b04e)

Reply:
I used only changeOscapProfile functions, other data comes from the rtk query

Reply:
Fair enough, would that mean that we don't need 
```typescript
    profileInformation: {
      profileId: OpenScap['profile_id'] | undefined;
      profileName: OpenScap['profile_name'];
      profileDescription: OpenScap['profile_description'];
    };
```
in the wizardSlice at all?

Together with `changeOscapProfileName` and `changeOscapProfileDescription` as I can't see those used outside of the wizardSlice as well.

Reply:
yes, I thought its good idea to add this inforamtion together with the profileInformation

Reply:
Got it, if we want to keep the information in the Wizard form so we for example don't have to make another API call on the Review step, then we absolutely can do that :) 

My only problem is now that we have the information structure in the store, but we don't fill it in anywhere so it stays empty.

Reply:
Good catch @regexowl! Yeah, keeping the bare minimum of data in the store is preferable.

Reply:
removed that part, thank you both